Sources/Amplitude/Amplitude.swift

```diff
@@ -290,6 +290,27 @@ public class Amplitude {
         timeline.apply(closure)
     }
 
+    public func openURL(_ url: NSURL) {
```

Comment:
are those 2 methods needed?

Reply:
Implementation is based on Amplitude-iOS code (https://github.com/amplitude/Amplitude-iOS/pull/455)
[Docs](https://amplitude.atlassian.net/wiki/spaces/GROWTH/pages/2109079759/Test+Mobile+SDK+Default+Events#iOS-Objective-C-SDK)

Reply:
I would also like to get rid of these functions and replace with a `OpenUrlEvent` that can be tracked with `track()` like all other events. See comment about screen() above. cc @liuyang1520

Reply:
Overload `track` sounds good to me, good idea!

Reply:
1. Added `DeepLinkOpenedEvent`. Let me know if it should be renamed.
2. Removed `deepLink` from `DefaultTrackingOptions` - currently the property has no mush sense